docs/getting_started/tutorial.md

```diff
@@ -405,23 +434,30 @@ Create a new template file at the location `blog/templates/blog/blog_page.html`.
 Note the use of Wagtail's built-in `get_parent()` method to obtain the
 URL of the blog this post is a part of.
 
-Now create a few blog posts as children of `BlogIndexPage`.
-Be sure to select the type "Blog Page" when creating your posts.
+Now, go to your [Admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface) and create a few blog posts as children of `BlogIndexPage` by following these steps:
+1. Click **Pages** from the Wagtail [Side bar](https://guide.wagtail.org/en-latest/how-to-guides/find-your-way-around/#the-sidebar), and then click **Home**
```

Comment:
For this next set of sub-steps. I recommend something like a bold small title for each. 

Similar to what you have for number three but bolded or with a heading. 

Having a mix of numbered items and content is a bit tricky with markdown and also a bit hard to read.